### ISSUE TITLE
About AVA

### ISSUE THREAD

**Issue by huboyang040911** [open-mmlab/mmaction2#2917]
### The doc issue

I recorded the videos myself and built my own AVA dataset following the official guidelines, which includes two action categories (hit, sit). The configuration file I used is **_slowonly_kinetics400-pretrained-r50_8xb16-4x16x1-20e_ava21-rgb.py_** in the directory of mmaction2/configs/detection/slowfast. However, I ran into an issue during training: **_all mAP values are 0_**, while the loss is actually decreasing. Could anyone help me with this problem?

我自己录制了视频并按照官方说明构造了自己的AVA数据集，一共有两个动作类别（hit，sit），使用的配置文件是mmaction2/configs/detection/slowfast下的slowonly_kinetics400-pretrained-r50_8xb16-4x16x1-20e_ava21-rgb.py，但我在训练时遇到了问题：所有的mAP都是0，但损失是有在下降的，有人能帮帮我吗？
